examples/pure/src/lib.rs

```diff
@@ -250,6 +250,48 @@ fn default_value(num: Number) -> Number {
     num
 }
 
+#[gen_stub_pyfunction]
+#[pyfunction]
+#[gen_stub(override_type(type_repr="collections.abc.Callable[[str]]", imports=("collections.abc")))]
```

Comment:
I think it's not clear at first glance that this overrides the return type. To be more explicit, I'd like to use a name like `override_return_type` instead of `override_type`.